tests/testAutopas/tests/tuning/searchSpace/EvidenceCollectionTest.cpp

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file EvidenceCollectionTest.cpp
+ * @author F. Gratl
+ * @date 05.07.23
+ */
+
+#include "EvidenceCollectionTest.h"
+
+// TODO
```

Comment:
Would be fitting for this PR, but we can also just open an issue here to not forget.